packages/dialect-react-ui/index.css

```diff
@@ -1,11 +1,11 @@
 /* Prefixed preflight */
 .dialect {
```

Comment:
do we still need .dialect?

Reply:
preflight is not prefixed, so yeah, unfortunately, we need it

Reply:
Sorry, what I meant was that since we have prefixes now, I think we don't need .dialect anymore. Wdyt?

Reply:
yeah, if I got the question right it's about the next comment. The thing is if we add `.dialect` on the preflight, it increases specificity of the preflight. So for example, if we have in preflight

```css 
.dialect button { 
  padding: 0; 
}
```
and we use class to add padding,

```jsx 
<button className="px-3" />
```

the prefixed intial styles will be applied.

So there are two ways to resolve:
1. Prefix all styles via `important: '.dialect'`
2. Use classes like `dt-button` instead of prefixed preflight 

After a closer look, It seems to me the later is the only way to go. Since the variable overrides classes need the use of `!important` to overcome `.dialect` specifity also.